src/storm/modelchecker/abstraction/AbstractAbstractionRefinementModelChecker.cpp

```diff
@@ -362,7 +362,7 @@ std::pair<std::unique_ptr<CheckResult>, std::unique_ptr<CheckResult>> AbstractAb
 template<typename ModelType>
 std::pair<std::unique_ptr<CheckResult>, std::unique_ptr<CheckResult>> AbstractAbstractionRefinementModelChecker<ModelType>::computeQuantitativeResult(
     Environment const& env, storm::models::symbolic::Dtmc<DdType, ValueType> const& abstractModel,
-    storm::abstraction::SymbolicStateSet<DdType> const& constraintStates, storm::abstraction::SymbolicStateSet<DdType> const& targetStates,
+    [[maybe_unused]] storm::abstraction::SymbolicStateSet<DdType> const& constraintStates, storm::abstraction::SymbolicStateSet<DdType> const& targetStates,
```

Comment:
Why is this okay that this is not used?

Reply:
That's an error just like the other two spots in this class you mentioned. I thought that this class was part of an inheritance / override situation with the GameBasedMdpModelChecker, but turns out they just have functions with the same names. Will be removing that again!